core/user-flow.js

```diff
@@ -221,16 +195,41 @@ class UserFlow {
   }
 
   /**
-   * @return {LH.UserFlow.FlowArtifacts}
+   * @return {Promise<LH.UserFlow.FlowArtifacts>}
    */
-  createArtifactsJson() {
+  async createArtifactsJson() {
     return {
```

Comment:
why is this async

Reply:
If we needed to make this async in the future it would need to be done in a breaking change. The API refresh in this PR seemed like a good time to protect ourselves.

I'm fine leaving this out though, and circling back if we actually need it.

Reply:
Can you give an hypothetical for when this might need to be async? My preference would be to not make this async  but if you think it's likely this needs to be a promise later, then ok

Reply:
Not really. In theory all the artifacts should be stored on the flow object when this function is called. I'll remove the `async`